dss-api.yml

```diff
@@ -126,6 +126,73 @@ definitions:
       - uuid
       - name
       - versions
+  blob_common_metadata:
+    type: object
+    description: Common fields between the files list of a bundle and the metadata tracking an unique file.
+    properties:
+      uuid:
+        type: string
+        description: File unique ID
+      timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of file creation measured in milliseconds since epoch
```

Comment:
Are we doing milliseconds or ISO timestamps? In the spec I think I saw ISO timestamps. I'm on the fence with maybe a little bit of preference for ISO timestamps since they're human readable and more type inferable.